packages/snip20/src/handle.rs

```diff
@@ -0,0 +1,249 @@
+use std::fmt;
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use cosmwasm_std::{Uint128, HumanAddr, Binary};
+use cosmwasm_utils::viewing_key::ViewingKey;
+
+use crate::data::ContractStatusLevel;
+
+macro_rules! generate_disable_msg {
```

Comment:
how does this work

Reply:
1. Done

2. Hey, its there if you need it, it doesn't break anything. More importantly, I don't want us having to create another fork/copy and wondering how to keep it in sync the next time a single method needs to be disabled in a token. That way is the most configurable and doesn't really cost anything.

3. I don't really see a point in that because its only type declarations which reference each other. I don't think it will actually help with anything. But if we are to do it, let's not do it now as it is not urgent.